@navikt/core/css/layout-container.css

```
@@ -0,0 +1,27 @@
+.navds-layout-container {
+  padding-left: var(--ac-l-container-padding-xs, var(--navds-spacing-4));
+  padding-right: var(--ac-l-container-padding-xs, var(--navds-spacing-4));
+  margin: 0 auto;
```

Siden padding settes på høyre og venstre, tenker jeg det er ryddig å bare sette margin sånn også.  
```suggestion
  margin-left: auto;
  margin-right: auto;
```